LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -46,17 +46,20 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
     var glucoseUnit: HKUnit
     var supportedModelSettings: SupportedInsulinModelSettings
     let appName: String
+    let mode: PresentationMode
     
     public init(
         viewModel: InsulinModelSelectionViewModel,
         glucoseUnit: HKUnit,
         supportedModelSettings: SupportedInsulinModelSettings,
-        appName: String
+        appName: String,
+        mode: PresentationMode = .settings
```

Comment:
Add this `mode` in order to not wrap the screen in a `NavigationView` if we're using this as part of the acceptance flow